app/globals.css

```diff
@@ -1,5 +1,74 @@
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+@tailwind base;
+
+
+@layer base {
+  body {
+    @apply bg-light-background dark:bg-dark-background text-light-text dark:text-dark-text w-full p-2 m-auto;
+  }
+  
+  main div {
+    @apply flex flex-col items-center justify-center m-auto;
+  }
+
+  h1 {
+    @apply font-serif font-semibold text-3xl;
+  }
+
+  .button-primary {
+    @apply bg-[#ffa51b] shadow-md my-2 m-auto rounded-full p-2;
+  }
+
+  div:has( > label) {
+    @apply w-fit;
+  }
+
+  .label-primary {
+    @apply font-sans text-sm font-semibold text-left mt-2 w-full pl-2;
+  }
+
+  .input-primary {
+    @apply rounded-lg border border-black px-2;
+  }
+
+  footer {
+    @apply text-sm;
+  }
+
+  .line hr {
+    @apply flex-1 mx-1;
+  }
+
+  @media screen and (max-width: 390px) {
+    body {
+      @apply bg-[#F4FBF9] m-auto text-black min-w-[200px];
```

Comment:
I'm just catching onto the fact that we are specifically overriding dark mode for "mobile" users 😅 

We might consider increasing the `max-width` for this media query to `430px` to be consistent with larger phones.